doc/en/components/grids/_shared/display-density.md

```diff
@@ -756,12 +737,16 @@ We can now extend our sample and add `RowHeight` property to the `{ComponentName
 
  ```razor
  <{ComponentSelector} Width="100%" Height="100%"
-             @ref=grid
-             AutoGenerate=true
-             Data=northwindEmployees
-             RowHeight="80"
-             DisplayDensity=@density>
+             @ref="grid"
+             AutoGenerate="true"
+             Data="northwindEmployees"
+             RowHeight="rowHeight"
+             DisplayDensity="@density">
 </{ComponentSelector}>
+
+@code {
+    private string rowHeight = "80px";
+}
  ```
 
  <!-- WebComponents -->
```

Comment:
The code snippet bellow is wrong. `display-density="Cosy"` is wrong, should be display-density="cosy"

Reply:
And there is one more occurrence of it in this doc